### ISSUE TITLE
Formly Material Checkbox can't find '_inputElement' onDestroy in v5.10.7

### ISSUE THREAD

**Issue by Gizrah** [ngx-formly/ngx-formly#2620]
**Description**
After updating to 5.10.7, any form I open without a checkbox will say give the following error:

```
ERROR TypeError: Cannot read property '_inputElement' of undefined
    at FormlyFieldCheckbox.ngOnDestroy (ngx-formly-material-checkbox.js:91)
    at executeOnDestroys (core.js:7113)
    at cleanUpView (core.js:7026)
    at destroyViewTree (core.js:6859)
    at destroyLView (core.js:7004)
    at RootViewRef.destroy (core.js:22299)
    at ComponentRef$1.destroy (core.js:24842)
    at FormlyConfig.resolveFieldTypeRef (ngx-formly-core.js:561)
    at CoreExtension.getFieldComponentInstance (ngx-formly-core.js:3416)
    at CoreExtension.prePopulate (ngx-formly-core.js:3239)
```

**Minimal Reproduction**
https://stackblitz.com/edit/ngx-formly-ui-material-69mzdc

**Your Environment**

- Angular version: 11.0.4 -- also present in 10.x on example
- Formly version: 5.10.7

**Additional context**
While the form won't kill itself using ng serve, when hosting a prod build it will break completely.

**Comment by aitboudad** [comment]
fixed in `5.10.8`

**Comment by hkadim** [comment]
Repro in ngAfterViewInit 
Angular version: 11.2.12 
Formly version: 5.10.19
ERROR TypeError: Cannot read property '_inputElement' of undefined
    at FormlyFieldCheckboxComponent.ngAfterViewInit (ngx-formly-material-checkbox.js:58)
